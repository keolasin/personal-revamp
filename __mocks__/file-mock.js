module.exports = {
	thumnailImg: {
		childImageSharp: {
			fluid: {
				aspectRatio: 1,
				src: "/static/adc26bdf4e25d6796d9fe999ba7cec1a/14b42/portrait.jpg",
				srcSet: "/static/adc26bdf4e25d6796d9fe999ba7cec1a/f836f/portrait.jpg 200w,\n/static/adc26bdf4e25d6796d9fe999ba7cec1a/2244e/portrait.jpg 400w,\n/static/adc26bdf4e25d6796d9fe999ba7cec1a/14b42/portrait.jpg 800w,\n/static/adc26bdf4e25d6796d9fe999ba7cec1a/47498/portrait.jpg 1200w,\n/static/adc26bdf4e25d6796d9fe999ba7cec1a/0e329/portrait.jpg 1600w,\n/static/adc26bdf4e25d6796d9fe999ba7cec1a/70963/portrait.jpg 3456w",
				srcWebp: "/static/adc26bdf4e25d6796d9fe999ba7cec1a/58556/portrait.webp",
				srcSetWebp: "/static/adc26bdf4e25d6796d9fe999ba7cec1a/61e93/portrait.webp 200w,\n/static/adc26bdf4e25d6796d9fe999ba7cec1a/1f5c5/portrait.webp 400w,\n/static/adc26bdf4e25d6796d9fe999ba7cec1a/58556/portrait.webp 800w,\n/static/adc26bdf4e25d6796d9fe999ba7cec1a/99238/portrait.webp 1200w,\n/static/adc26bdf4e25d6796d9fe999ba7cec1a/7c22d/portrait.webp 1600w,\n/static/adc26bdf4e25d6796d9fe999ba7cec1a/cadd6/portrait.webp 3456w",
				sizes: "(max-width: 800px) 100vw, 800px",
				originalImg: "/static/adc26bdf4e25d6796d9fe999ba7cec1a/70963/portrait.jpg"
			}
		}
	},
	mobileImage: {
		childImageSharp: {
			fluid: {
				aspectRatio: 0.7987220447284346,
				src: "/static/b3bb68c33b441137e56536ad52f48d32/724c8/jTree.jpg",
				srcSet:
					"/static/b3bb68c33b441137e56536ad52f48d32/84d81/jTree.jpg 250w,\n/static/b3bb68c33b441137e56536ad52f48d32/f0719/jTree.jpg 500w,\n/static/b3bb68c33b441137e56536ad52f48d32/724c8/jTree.jpg 1000w,\n/static/b3bb68c33b441137e56536ad52f48d32/d79bd/jTree.jpg 1500w,\n/static/b3bb68c33b441137e56536ad52f48d32/a66ad/jTree.jpg 2000w,\n/static/b3bb68c33b441137e56536ad52f48d32/5bdee/jTree.jpg 3456w",
				srcSetWebp:
					"/static/b3bb68c33b441137e56536ad52f48d32/1d872/jTree.webp 250w,\n/static/b3bb68c33b441137e56536ad52f48d32/4e6d4/jTree.webp 500w,\n/static/b3bb68c33b441137e56536ad52f48d32/36ebb/jTree.webp 1000w,\n/static/b3bb68c33b441137e56536ad52f48d32/fd45d/jTree.webp 1500w,\n/static/b3bb68c33b441137e56536ad52f48d32/6e77b/jTree.webp 2000w,\n/static/b3bb68c33b441137e56536ad52f48d32/fa3d3/jTree.webp 3456w",
				sizes: "(max-width: 1000px) 100vw, 1000px",
				originalImg: "/static/b3bb68c33b441137e56536ad52f48d32/5bdee/jTree.jpg",
			},
		},
	},
	desktopImage: {
		childImageSharp: {
			fluid: {
				aspectRatio: 1.3966480446927374,
				src: "/static/a4cd4b58bfdb8357ae84b4cb5b5c7a1a/724c8/hiking.jpg",
				srcSet:
					"/static/a4cd4b58bfdb8357ae84b4cb5b5c7a1a/84d81/hiking.jpg 250w,\n/static/a4cd4b58bfdb8357ae84b4cb5b5c7a1a/f0719/hiking.jpg 500w,\n/static/a4cd4b58bfdb8357ae84b4cb5b5c7a1a/724c8/hiking.jpg 1000w,\n/static/a4cd4b58bfdb8357ae84b4cb5b5c7a1a/d79bd/hiking.jpg 1500w,\n/static/a4cd4b58bfdb8357ae84b4cb5b5c7a1a/a66ad/hiking.jpg 2000w,\n/static/a4cd4b58bfdb8357ae84b4cb5b5c7a1a/a521d/hiking.jpg 4608w",
				srcSetWebp:
					"/static/a4cd4b58bfdb8357ae84b4cb5b5c7a1a/1d872/hiking.webp 250w,\n/static/a4cd4b58bfdb8357ae84b4cb5b5c7a1a/4e6d4/hiking.webp 500w,\n/static/a4cd4b58bfdb8357ae84b4cb5b5c7a1a/36ebb/hiking.webp 1000w,\n/static/a4cd4b58bfdb8357ae84b4cb5b5c7a1a/fd45d/hiking.webp 1500w,\n/static/a4cd4b58bfdb8357ae84b4cb5b5c7a1a/6e77b/hiking.webp 2000w,\n/static/a4cd4b58bfdb8357ae84b4cb5b5c7a1a/6bce6/hiking.webp 4608w",
				sizes: "(max-width: 1000px) 100vw, 1000px",
				originalImg:
					"/static/a4cd4b58bfdb8357ae84b4cb5b5c7a1a/a521d/hiking.jpg",
			},
		},
	},
};
