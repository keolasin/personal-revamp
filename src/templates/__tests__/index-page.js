import React from "react";
import { render, screen } from "@testing-library/react";
import { useStaticQuery } from "gatsby";

import IndexPage from "../index-page";

// Mocks and fixate generate className
import { mobileImage, desktopImage, thumbnailImg } from "../../../__mocks__/file-mock.js";
jest.mock("short-uuid");

beforeEach(() => {
	useStaticQuery.mockImplementation(() => ({
		mobileImage,
		desktopImage,
        thumbnailImg,
        site: {
            siteMetadata: {
                title: `MReyes`,
                description: `Personal portfolio and photography site featuring the work of Matthew Reyes`,
                author: `@keolasin`,
            }
        }
	}));

	const uuid = require("short-uuid");
	uuid.generate.mockImplementation(() => "73WakrfVbNJBaAmhQtEeDv");
});

describe("Index Page", () => {
	// mock data
    // templateKey, title, thumbnailImg, heading, body
	const data = {
		markdownRemark: {
			html: "<p>Test body</p>",
			frontmatter: {
				title: "Index page",
				heading: "Test header",
				thumbnailImg: "testImg.jpg",
			},
            childrenFile: [
                {
                    childImageSharp: {
                        fluid: {
                            aspectRatio: 1,
                            base64: "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAUABQDASIAAhEBAxEB/8QAGAABAQEBAQAAAAAAAAAAAAAAAAQFAgP/xAAWAQEBAQAAAAAAAAAAAAAAAAAAAQL/2gAMAwEAAhADEAAAAa5NLNxv3EUxlnYP/8QAGhAAAgMBAQAAAAAAAAAAAAAAAREAAgMxIv/aAAgBAQABBQLTlStC2QxVDc3D7D6KE//EABYRAQEBAAAAAAAAAAAAAAAAAAIQMf/aAAgBAwEBPwE5DP/EABYRAQEBAAAAAAAAAAAAAAAAAAEQMf/aAAgBAgEBPwF2M//EABoQAAICAwAAAAAAAAAAAAAAAAERABACITH/2gAIAQEABj8C6oAMm73Rjr//xAAZEAEAAwEBAAAAAAAAAAAAAAABABExIRD/2gAIAQEAAT8h4OK9YL+NC35C6wM5KzyMWhBWy7Sf/9oADAMBAAIAAwAAABCwAH3/xAAYEQADAQEAAAAAAAAAAAAAAAAAARExcf/aAAgBAwEBPxCW4uGGU//EABgRAQEAAwAAAAAAAAAAAAAAAAEAEBFh/9oACAECAQE/ECgk7jov/8QAGxABAAIDAQEAAAAAAAAAAAAAAQARITFBYXH/2gAIAQEAAT8QCbVD2FOojJqTwTf2IKOAHkQKmXBYUSyqW2DgE87BJwvUDiN1P//Z",
                            originalImg: "/static/b1cb549c6d70b9d75525fb6e7fb3ac08/06655/portrait.jpg",
                            src: "/static/b1cb549c6d70b9d75525fb6e7fb3ac08/724c8/portrait.jpg",
                            srcSet: "/static/b1cb549c6d70b9d75525fb6e7fb3ac08/84d81/portrait.jpg 250w,\n/static/b1cb549c6d70b9d75525fb6e7fb3ac08/f0719/portrait.jpg 500w,\n/static/b1cb549c6d70b9d75525fb6e7fb3ac08/724c8/portrait.jpg 1000w,\n/static/b1cb549c6d70b9d75525fb6e7fb3ac08/d79bd/portrait.jpg 1500w,\n/static/b1cb549c6d70b9d75525fb6e7fb3ac08/06655/portrait.jpg 1600w"
                        }
                    }
                }
            ]
		}
	};

	it("renders data correctly", () => {
		const { container, getByText } = render(<IndexPage data={data} />);
        expect(getByText('Test body')).toBeInTheDocument();
	});

    it("matches the snapshot", () => {
        const { container } = render(<IndexPage data={data} />);
        expect(container).toMatchSnapshot();
    });
});
