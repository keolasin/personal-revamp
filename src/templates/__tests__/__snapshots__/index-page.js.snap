// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page matches the snapshot 1`] = `
.emotion-13 {
  width: 100%;
  min-height: 100vh;
  background-color: transparent;
}

.emotion-4 {
  font-family: "astounder-squared-bb",sans-serif;
  font-size: 62.5%;
  -webkit-text-decoration: none;
  text-decoration: none;
  text-align: center;
  margin: 0 auto;
  place-items: center center;
  width: 100%;
  min-height: 10vh;
  display: grid;
  grid-template-columns: 1fr 1fr 1fr;
  grid-template-rows: 1fr 1fr;
  grid-template-areas: ". home ." "photos projects contact";
  background-color: rgba(0,0,0,0.75);
  border-radius: 0px 0px 25px 25px;
}

@media (min-width:320px) {
  .emotion-4 {
    font-size: 1rem;
  }
}

@media (min-width:480px) {
  .emotion-4 {
    font-size: 1.4rem;
  }
}

@media (min-width:768px) {
  .emotion-4 {
    font-size: 1.6rem;
  }
}

@media (min-width:992px) {
  .emotion-4 {
    font-size: 1.8rem;
  }
}

@media (min-width:1200px) {
  .emotion-4 {
    font-size: 3rem;
  }
}

.emotion-4 a {
  color: #dbe7fb;
  -webkit-text-stroke-width: 1px;
  -webkit-text-stroke-color: black;
}

.emotion-4 a:hover {
  color: #798be4;
}

.emotion-4 a:focus {
  color: #bc9612;
}

.emotion-4 a:active {
  color: #bc9612;
}

@media (min-width:1200px) {
  .emotion-4 {
    width: 80%;
    grid-template-columns: 1fr 1fr 1fr 1fr;
    grid-template-rows: 1fr;
    grid-template-areas: "home photos projects contact";
  }
}

.emotion-0 {
  font-family: "astounder-squared-bb",sans-serif;
  font-size: 62.5%;
  -webkit-text-decoration: none;
  text-decoration: none;
  text-align: center;
  margin: 0 auto;
  font-size: 1.8rem;
  grid-area: home;
  margin-top: 5px;
}

@media (min-width:320px) {
  .emotion-0 {
    font-size: 1rem;
  }
}

@media (min-width:480px) {
  .emotion-0 {
    font-size: 1.4rem;
  }
}

@media (min-width:768px) {
  .emotion-0 {
    font-size: 1.6rem;
  }
}

@media (min-width:992px) {
  .emotion-0 {
    font-size: 1.8rem;
  }
}

@media (min-width:1200px) {
  .emotion-0 {
    font-size: 3rem;
  }
}

@media (min-width:320px) {
  .emotion-0 {
    font-size: 1.8rem;
    margin-top: 5px;
  }
}

@media (min-width:480px) {
  .emotion-0 {
    font-size: 2rem;
    margin-top: 10px;
    margin-bottom: 5px;
  }
}

@media (min-width:768px) {
  .emotion-0 {
    font-size: 2.3rem;
    margin-top: 10px;
    margin-bottom: 5px;
  }
}

@media (min-width:992px) {
  .emotion-0 {
    font-size: 2.5rem;
  }
}

@media (min-width:1200px) {
  .emotion-0 {
    font-size: 3.5rem;
    margin-top: 20px;
    margin-bottom: 22px;
  }
}

.emotion-1 {
  font-family: "astounder-squared-bb",sans-serif;
  font-size: 62.5%;
  -webkit-text-decoration: none;
  text-decoration: none;
  text-align: center;
  margin: 0 auto;
  font-size: 1.4rem;
}

@media (min-width:320px) {
  .emotion-1 {
    font-size: 1rem;
  }
}

@media (min-width:480px) {
  .emotion-1 {
    font-size: 1.4rem;
  }
}

@media (min-width:768px) {
  .emotion-1 {
    font-size: 1.6rem;
  }
}

@media (min-width:992px) {
  .emotion-1 {
    font-size: 1.8rem;
  }
}

@media (min-width:1200px) {
  .emotion-1 {
    font-size: 3rem;
  }
}

@media (min-width:480px) {
  .emotion-1 {
    font-size: 1.6rem;
  }
}

@media (min-width:768px) {
  .emotion-1 {
    font-size: 1.8rem;
  }
}

@media (min-width:992px) {
  .emotion-1 {
    font-size: 2rem;
  }
}

@media (min-width:1200px) {
  .emotion-1 {
    font-size: 2.5rem;
  }
}

.emotion-12 {
  font-family: "astounder-squared-bb",sans-serif;
  font-size: 62.5%;
  -webkit-text-decoration: none;
  text-decoration: none;
  text-align: center;
  margin: 0 auto;
  overflow-y: auto;
  height: 85vh;
  -webkit-scrollbar-width: none;
  -moz-scrollbar-width: none;
  -ms-scrollbar-width: none;
  scrollbar-width: none;
  -ms-verflow-style: none;
}

@media (min-width:320px) {
  .emotion-12 {
    font-size: 1rem;
  }
}

@media (min-width:480px) {
  .emotion-12 {
    font-size: 1.4rem;
  }
}

@media (min-width:768px) {
  .emotion-12 {
    font-size: 1.6rem;
  }
}

@media (min-width:992px) {
  .emotion-12 {
    font-size: 1.8rem;
  }
}

@media (min-width:1200px) {
  .emotion-12 {
    font-size: 3rem;
  }
}

.emotion-12::-webkit-scrollbar {
  display: none;
}

@media (min-width:768px) {
  .emotion-12 {
    width: 95%;
  }
}

@media (min-width:992px) {
  .emotion-12 {
    width: 90%;
  }
}

@media (min-width:1200px) {
  .emotion-12 {
    width: 80%;
  }
}

.emotion-10 {
  border-radius: 25px 25px 0px 0px;
  background-color: rgba(0,0,0,0.75);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column;
  -ms-flex-flow: column;
  flex-flow: column;
  position: fixed;
  bottom: 0;
  left: 0;
  right: 0;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

@media (min-width:768px) {
  .emotion-10 {
    -webkit-flex-flow: row;
    -ms-flex-flow: row;
    flex-flow: row;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
  }
}

.emotion-6 {
  border-radius: 50%;
  object-fit: cover;
  display: none;
}

@media (min-width:320px) {
  .emotion-6 {
    width: 125px;
    height: 125px;
  }
}

@media (min-width:480px) {
  .emotion-6 {
    width: 150px;
    height: 150px;
  }
}

@media (min-width:768px) {
  .emotion-6 {
    width: 125px;
    height: 125px;
    display: inline-block;
  }
}

@media (min-width:992px) {
  .emotion-6 {
    width: 175px;
    height: 175px;
  }
}

@media (min-width:1200px) {
  .emotion-6 {
    width: 225px;
    height: 225px;
  }
}

.emotion-8 {
  border-radius: 25px;
  color: #dbe7fb;
  font-family: "acumin-pro",sans-serif;
  line-height: 1.2em;
  font-size: 1rem;
  padding: 5px;
  margin: auto 0;
}

@media (min-width:768px) {
  .emotion-8 {
    font-size: 1.2rem;
    margin: 10px;
  }
}

@media (min-width:1200px) {
  .emotion-8 {
    font-size: 1.7rem;
  }
}

<div>
  <section
    class="emotion-13 emotion-14 gbi--1561385816-73WakrfVbNJBaAmhQtEeDv"
    style="background-position: center; background-repeat: no-repeat; background-size: cover; background-color: transparent; position: relative; opacity: 0.99;"
  >
    <style>
      
          .emotion-13.emotion-14.gbi--1561385816-73WakrfVbNJBaAmhQtEeDv:before,
          .emotion-13.emotion-14.gbi--1561385816-73WakrfVbNJBaAmhQtEeDv:after {
            content: '';
            display: block;
            position: absolute;
            width: 100%;
            height: 100%;
            top: 0;
            left: 0;
            
            transition: opacity 0.5s ease 250ms;
            background-position: center;
background-repeat: no-repeat;
background-size: cover;
background-color: transparent;

          }
          .emotion-13.emotion-14.gbi--1561385816-73WakrfVbNJBaAmhQtEeDv:before {
            z-index: -100;
            
            
            opacity: 1; 
          }
          .emotion-13.emotion-14.gbi--1561385816-73WakrfVbNJBaAmhQtEeDv:after {
            z-index: -101;
            
            
            
          }
        
    </style>
    <noscript />
    <nav
      class="emotion-4 emotion-5"
    >
      <a
        class="emotion-0"
        href="/"
      >
        MReyes
      </a>
      <a
        class="emotion-1"
        href="/albums"
        style="grid-area: photos;"
      >
        Photos
      </a>
      <a
        class="emotion-1"
        href="/projects"
        style="grid-area: projects;"
      >
        Projects
      </a>
      <a
        class="emotion-1"
        href="/contact"
        style="grid-area: contact;"
      >
        Contact
      </a>
    </nav>
    <main
      class="emotion-12"
    >
      <section
        class="emotion-10 emotion-11"
      >
        <div
          class="emotion-6 emotion-7 gatsby-image-wrapper"
          style="position: relative; overflow: hidden;"
        >
          <div
            aria-hidden="true"
            style="width: 100%; padding-bottom: 100%;"
          />
          <img
            alt=""
            aria-hidden="true"
            src="data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAUABQDASIAAhEBAxEB/8QAGAABAQEBAQAAAAAAAAAAAAAAAAQFAgP/xAAWAQEBAQAAAAAAAAAAAAAAAAAAAQL/2gAMAwEAAhADEAAAAa5NLNxv3EUxlnYP/8QAGhAAAgMBAQAAAAAAAAAAAAAAAREAAgMxIv/aAAgBAQABBQLTlStC2QxVDc3D7D6KE//EABYRAQEBAAAAAAAAAAAAAAAAAAIQMf/aAAgBAwEBPwE5DP/EABYRAQEBAAAAAAAAAAAAAAAAAAEQMf/aAAgBAgEBPwF2M//EABoQAAICAwAAAAAAAAAAAAAAAAERABACITH/2gAIAQEABj8C6oAMm73Rjr//xAAZEAEAAwEBAAAAAAAAAAAAAAABABExIRD/2gAIAQEAAT8h4OK9YL+NC35C6wM5KzyMWhBWy7Sf/9oADAMBAAIAAwAAABCwAH3/xAAYEQADAQEAAAAAAAAAAAAAAAAAARExcf/aAAgBAwEBPxCW4uGGU//EABgRAQEAAwAAAAAAAAAAAAAAAAEAEBFh/9oACAECAQE/ECgk7jov/8QAGxABAAIDAQEAAAAAAAAAAAAAAQARITFBYXH/2gAIAQEAAT8QCbVD2FOojJqTwTf2IKOAHkQKmXBYUSyqW2DgE87BJwvUDiN1P//Z"
            style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; opacity: 1; transition-delay: 500ms;"
          />
          <picture>
            <source
              srcset="/static/b1cb549c6d70b9d75525fb6e7fb3ac08/84d81/portrait.jpg 250w,
/static/b1cb549c6d70b9d75525fb6e7fb3ac08/f0719/portrait.jpg 500w,
/static/b1cb549c6d70b9d75525fb6e7fb3ac08/724c8/portrait.jpg 1000w,
/static/b1cb549c6d70b9d75525fb6e7fb3ac08/d79bd/portrait.jpg 1500w,
/static/b1cb549c6d70b9d75525fb6e7fb3ac08/06655/portrait.jpg 1600w"
            />
            <img
              alt="Yours truly"
              loading="lazy"
              src="/static/b1cb549c6d70b9d75525fb6e7fb3ac08/724c8/portrait.jpg"
              srcset="/static/b1cb549c6d70b9d75525fb6e7fb3ac08/84d81/portrait.jpg 250w,
/static/b1cb549c6d70b9d75525fb6e7fb3ac08/f0719/portrait.jpg 500w,
/static/b1cb549c6d70b9d75525fb6e7fb3ac08/724c8/portrait.jpg 1000w,
/static/b1cb549c6d70b9d75525fb6e7fb3ac08/d79bd/portrait.jpg 1500w,
/static/b1cb549c6d70b9d75525fb6e7fb3ac08/06655/portrait.jpg 1600w"
              style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; opacity: 0; transition: opacity 500ms;"
            />
          </picture>
          <noscript>
            &lt;picture&gt;&lt;source srcset="/static/b1cb549c6d70b9d75525fb6e7fb3ac08/84d81/portrait.jpg 250w,
/static/b1cb549c6d70b9d75525fb6e7fb3ac08/f0719/portrait.jpg 500w,
/static/b1cb549c6d70b9d75525fb6e7fb3ac08/724c8/portrait.jpg 1000w,
/static/b1cb549c6d70b9d75525fb6e7fb3ac08/d79bd/portrait.jpg 1500w,
/static/b1cb549c6d70b9d75525fb6e7fb3ac08/06655/portrait.jpg 1600w" /&gt;&lt;img loading="lazy" srcset="/static/b1cb549c6d70b9d75525fb6e7fb3ac08/84d81/portrait.jpg 250w,
/static/b1cb549c6d70b9d75525fb6e7fb3ac08/f0719/portrait.jpg 500w,
/static/b1cb549c6d70b9d75525fb6e7fb3ac08/724c8/portrait.jpg 1000w,
/static/b1cb549c6d70b9d75525fb6e7fb3ac08/d79bd/portrait.jpg 1500w,
/static/b1cb549c6d70b9d75525fb6e7fb3ac08/06655/portrait.jpg 1600w" src="/static/b1cb549c6d70b9d75525fb6e7fb3ac08/724c8/portrait.jpg" alt="Yours truly" style="position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/&gt;&lt;/picture&gt;
          </noscript>
        </div>
        <h1
          class="emotion-8 emotion-9"
        >
          <p>
            Test body
          </p>
        </h1>
      </section>
    </main>
  </section>
</div>
`;
